### PR TITLE
[DOCS] Updating Checkpoint terms page

### PR DESCRIPTION
**Pull request by donaldheppner**
Updating the Checkpoint terms page, including the tests in checkpoints_and_actions.py

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: tests/integration/test_script_runner.py [hunk 1]
```diff
@@ -282,6 +276,12 @@
         data_dir="tests/test_sets/taxi_yellow_tripdata_samples",
         user_flow_script="tests/integration/docusaurus/reference/glossary/batch_request.py",
     ),
+    IntegrationTestFixture(
+        name="RNUME checkpoints_and_actions_core_concepts",
```
Comment (anthonyburdi) [inline tests/integration/test_script_runner.py]:
```suggestion
        name="RUNME checkpoints_and_actions_core_concepts",
```

File: tests/integration/docusaurus/reference/core_concepts/checkpoints_and_actions.py [hunk 2]
```diff
@@ -343,9 +339,9 @@
 ]
 
 assert first_expectation_suite == "my_expectation_suite"
-assert first_data_asset == "yellow_tripdata_sample_2019-01"
+assert first_data_asset == "taxi_asset"
 assert second_expectation_suite == "my_other_expectation_suite"
-assert second_data_asset == "yellow_tripdata_sample_2019-02"
+assert second_data_asset == "taxi_asset"
```
Comment (anthonyburdi) [inline tests/integration/docusaurus/reference/core_concepts/checkpoints_and_actions.py]: We could check `batch_identifiers` here to make sure the right batch was used for the validation (and other places we check the asset names). Something like the code snippet below. I also looked into batch_id, but this seems to be included in the results at the expectation level and not the validation level.

```
first_batch_identifiers = first_validation_result["meta"]["active_batch_definition"][
    "batch_identifiers"
]
assert first_batch_identifiers == {
    "path": "yellow_tripdata_sample_2019-01.csv",
    "year": "2019",
    "month": "01",
}

assert second_expectation_suite == "my_expectation_suite"
assert second_data_asset == "taxi_asset"
second_batch_identifiers = second_validation_result["meta"]["active_batch_definition"][
    "batch_identifiers"
]
assert second_batch_identifiers == {
    "path": "yellow_tripdata_sample_2019-02.csv",
    "year": "2019",
    "month": "02",
}
```

Comment (anthonyburdi) [reply tests/integration/docusaurus/reference/core_concepts/checkpoints_and_actions.py]: (Implemented in https://github.com/great-expectations/great_expectations/pull/7722/commits/63f728dcbc0cac01c8cf2a1f29848facc34b2e0e)

File: docs/docusaurus/docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md [hunk 3]
```diff
@@ -152,6 +113,37 @@ Alternatively, the above Python code can be embedded in your pipeline.
...
+## Additional notes
+
+This command will return posix status codes and print messages as follows:
+
+    +-------------------------------+-----------------+-----------------------+
+    | **Situation**                 | **Return code** | **Message**           |
+    +-------------------------------+-----------------+-----------------------+
+    | all validations passed        | 0               | Validation succeeded! |
+    +-------------------------------+-----------------+-----------------------+
+    | one or more validation failed | 1               | Validation failed!    |
```
Comment (anthonyburdi) [inline docs/docusaurus/docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md]: Nit
```suggestion
    | All validations passed        | 0               | Validation succeeded! |
    +-------------------------------+-----------------+-----------------------+
    | One or more validation failed | 1               | Validation failed!    |
```

File: tests/integration/test_script_runner.py [hunk 4]
```diff
@@ -282,12 +276,24 @@
...
         data_context_dir=None,
         data_dir="tests/test_sets/taxi_yellow_tripdata_samples",
         user_flow_script="tests/integration/docusaurus/validation/checkpoints/how_to_create_a_new_checkpoint.py",
     ),
+    # IntegrationTestFixture(
+    #     name="RUNME validate_data_by_running_a_checkpoint",
+    #     user_flow_script="tests/integration/docusaurus/validation/checkpoints/how_to_validate_data_by_running_a_checkpoint.py",
+    #     data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
+    #     data_dir="tests/test_sets/taxi_yellow_tripdata_samples/first_3_files",
+    # ),
```
Comment (anthonyburdi) [inline tests/integration/test_script_runner.py]: Any reason this is commented out?

**Review by anthonyburdi — COMMENTED** [review]
Looks good, just a nit and question

**Review by Kilo59 — APPROVED** [review]

**Pull request merged**